src/base/trafficGraph.js

```diff
@@ -756,8 +761,10 @@ class TrafficGraph extends EventEmitter {
 
     _.each(this.nodes, (n) => { n.filtered = false; });
     _.each(this.connections, (c) => { c.filtered = false; });
-    this._updateConnectionFilters(filters);
-    this._updateNodeFilters(filters);
+    if (Object.keys(this.nodes).length > 1) {
```

Comment:
@jrsquared, @Baransu,
So where would you use `const forceNodesVisible = Object.keys(this.nodes.length > 1`?   Is that to augment the `if (node.connected)`?

What I did when I chopped out the unconnected node filtering to keep the `totalNodes++` count accurate, was a simple job.  I did make one addition (that I don't actually use... yet) that seemed like a good idea for the future.
```javascript
// Up at the top of _relayout()
    let unconnectedNodes = 0;

// Down at 783 in https://github.com/Baransu/vizceral/blob/entry-node/src/base/trafficGraph.js
    _.each(this.nodes, (node) => {
      if (useInLayout(node)) {
        graph.nodes.push(node);
      }

      totalNodes++;
      if (node.isVisible()) { visibleNodes++; }
      if(node.connected) (unconnectedNodes++;}
    });

    this.nodeCounts.total = totalNodes;
    this.nodeCounts.visible = visibleNodes;
    this.nodeCounts.unconnected = unconnectedNodes;
```